src/CreepLaw/NonLinearPeierlsCreep.jl

```diff
@@ -53,8 +53,8 @@ struct NonLinearPeierlsCreep{T,N,U1,U2,U3,U4,U5} <: AbstractCreepLaw{T}
         q=1.0NoUnits,
         o=0.5NoUnits,
         TauP=8.5e9Pa,
-        A=5.7e11MPa^(-2) * s^(-1),
-        E=476.0kJ / mol,
+        A=0.57Pa^(-2) * s^(-1),
```

Comment:
0.57Pa isn't 5.7e11
```suggestion
        A=5.7e17Pa^(-2) * s^(-1),
```

Reply:
As per README @ GeoParams:
```Julia
julia> using GeoParams;
julia> A=5.7e11MPa^(-2) * s^(-1)
5.7e11 MPa⁻²·⁰ s⁻¹·⁰
julia> uconvert(Pa^-2*s^-1, A)
0.57 Pa⁻²·⁰ s⁻¹·⁰
```
I guess every geodynamicists must have fallen into this trap at least once.